apps/client/src/app/modules/list/list-panel/list-panel.component.ts

```diff
@@ -38,6 +38,10 @@ import { TeamcraftComponent } from '../../../core/component/teamcraft-component'
 })
 export class ListPanelComponent extends TeamcraftComponent {
 
+  //Bounds for number of a single item in a list
+  maxAmount = 999;
```

Comment:
999 is way too low, In my opinion, if you want a maximum, you have to make it at least 10^9 because technically, you can even list gils in lists.

Even without considering gils, a lot of lists actually have 1000+ of a single item listed, for stuff like pre-ultimate consumables prep.

Reply:
Should I apply a limit to 10^9, or find a way to not apply any max limit for this field ?

Reply:
How did you manage to reach infinity using scroll wheel? Because I am really wondering how it is even possible, considering chrome handles numbers > 10^32

Reply:
I have never reached Inifinity using scroll wheel (yes, it's infinity so .. !), but i put this max bounds to be coherent with the other bounds on search-result.component.ts. I approve to put a limit to prevent user creating list with foolish amounts of items on it. You told me that test on Beta serv with millions of items doesn't create any crash of the application, so maybe we can just remove the max limit !